src/app/public/modules/card/card.component.ts

```diff
@@ -38,6 +40,9 @@ export class SkyCardComponent implements AfterContentInit, OnDestroy {
   @ContentChildren(SkyCardTitleComponent)
   public titleComponent: QueryList<SkyCardTitleComponent>;
 
+  @ContentChildren(SkyInlineDeleteComponent)
+  public inlineDeleteComponent: QueryList<SkyInlineDeleteComponent>;
```

Comment:
`inlineDeleteComponent` should be ordered before `titleComponent`.

Reply:
Done